product_docs/docs/tde/15/enabling_tde_pg_upgrade.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: "Enabling TDE with pg_upgrade"
+navTitle: Enabling TDE with pg_upgrade
+---
+
+EDB supports using [pg_upgrade](/tde/latest/pg_upgrade/) to add encryption to unencrypted systems. 
+This utility allows you to enable TDE when performing an upgrade to a more recent version, when transitioning from an open-source 
+postgreSQL instance to an EDB-supported instance, or to enable encryption while maintaining your current instance.
+
+You can also use `pg_upgrade` on already encrypted systems to rotate the data encryption keys. 
```

Comment:
Note to self: Add cross-linking from and to https://www.enterprisedb.com/docs/tde/latest/key_stores/#key-rotation